.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,55 @@
+name: Prepare release
+on:
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: new release version
+        required: true
+        default: (for example, 0.10.5)
+
+jobs:
+  release:
+    name: Prepare release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2.3.4
+
+      - name: Check input
+        run: |
+          if [[ ! "${{ github.event.inputs.release_version }}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            echo 'version "${{ github.event.inputs.release_version }}" not in ###.###.### format'
+            exit 1
+          fi
+          if [[ $( git ls-remote --tags origin refs/tags/v${{ github.event.inputs.release_version }} ) ]]; then
+            echo 'Tag "v${{ github.event.inputs.release_version }}" already exists. Double-check the version to release!'
+            exit 1
+          fi
+
+      - name: Create tag for release and prepare branch for next development version
+        run: |
+          git checkout -b release-v${{ github.event.inputs.release_version }}
+          git config user.email ${{ github.actor }}@users.noreply.github.com
```

Comment:
This email would make a bad commit history. If this is the only way, then it's okay.

Reply:
I believe this is customary. For example, dependabots:
```
commit 8c0de2dc7dfc3749d6ea920725c69ca9b20183fa
Author: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Date:   Mon Aug 2 14:59:06 2021 -0400

    Bump commons-compress from 1.20 to 1.21 (#875)
```

Reply:
Good. Okay.